phpBB/phpbb/feed/controller/feed.php

```diff
@@ -295,6 +295,7 @@ protected function send_feed_do(feed_interface $feed)
 	{
 		$feed_updated_time = 0;
 		$item_vars = array();
+		$this->user->add_lang('viewtopic');
```

Comment:
It's better to inject `language` service dependency and use `$this->language->add_lang()` instead because language methods are to be removed in phpBB 4.0..

Reply:
> It's better to inject `language` service dependency and use `$this->language->add_lang()` instead because language methods are to be removed from `user` service in phpBB 4.0.

Thanks a lot!